best-practices/README.md

```diff
@@ -101,6 +101,10 @@ Rails
   environment variables are detected on deploy.
 * [Use blocks][date-block] when declaring date and time attributes in FactoryGirl factories.
 * Use `touch: true` when declaring `belongs_to` relationships.
+* Name service classes after the action they perform
```

Comment:
Pretty much yes, just a different naming concept. Both seem to utilize the [Command Pattern](https://en.wikipedia.org/wiki/Command_pattern).